lib/api/include/irods/get_resource_info_for_operation.h

```diff
@@ -0,0 +1,31 @@
+#ifndef IRODS_GET_RESOURCE_INFO_FOR_OPERATION_H
+#define IRODS_GET_RESOURCE_INFO_FOR_OPERATION_H
+
```

Comment:
Replace this line with the following.
```cpp

/// file

```

That instructs doxygen to inspect this file for documentation.